src/views/Claim/components/VestingNotice.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { Box, Button, Notice, NoticeContent, NoticeIcon, Spacer } from "react-neu";
+import styled from "styled-components";
+
+const VestingNotice: React.FC = () => (
```

Comment:
we dont want duplicated components. and as of now should move this component to the global components folder